tests/common.py

```diff
@@ -6,8 +6,8 @@
 TEST_PATH = os.path.dirname(os.path.abspath(__file__))
 
 # Set path for executable:
-DUNE_EXE = os.path.split(TEST_PATH)[0] + "/dune"
-print("Executable path: ", DUNE_EXE)
+DUNES_EXE = os.path.split(TEST_PATH)[0] + "/dune"
```

Comment:
I'm pretty sure the tests can't pass without this change! :-) 
```suggestion
DUNES_EXE = os.path.split(TEST_PATH)[0] + "/dunes"
```

Reply:
Renamed.